docs/design-language.md

```diff
@@ -225,6 +225,53 @@ Panels come with an assortment of configurable navigation controls:
 
 ### Examples
 
-<img src="/img/panels/panel-welcome.jpg" alt="Example of an app 'Welcome Panel'" title="App 'Welcome Panel'" width="435" height="513" class="img-example" />
-<img src="/img/panels/panel-feature.jpg" alt="Example of the 'Result/Feature Details' panel" title="'Result/Feature Details' panel" width="435" height="513" class="img-example" />
+<img src="/img/panels/panel-welcome.jpg" alt="Welcome Dialog Panel" title="Welcome Dialog Panel" width="415" height="493" class="img-example" /> <img src="/img/panels/panel-feature.jpg" alt="Features Panel" title="Features Panel" width="415" height="493" class="img-example" />
+
+## Dialogs
+
+### Modal Dialogs
+
+Occasionally, there will be a need to display information, warn the user, or present controls in a manner that needs to utilize a more focused interface. Modal dialogs provide a flexible layout strategy to allow the user to put any content they want before the user and draws their focus to it, setting the rest of the app into a non-interactive state until it is dismissed. This allows important information to be displayed, and preserves the background information.
+
+### Examples
+
+<img src="/img/modals/measurement-units.jpg" alt="Units under the measurement toolbar" title="Units under the measure toolbar" width="444" height="228" class="img-example"/>
+
+<img src="/img/modals/polygon-styles.jpg" alt="Polygon Styles under the draw menu" title="Polygon Styles under the draw menu" width="600" height="530" class="img-example" />
+
+<img src="/img/modals/share-my-map.jpg" alt="Share my Map" title="Share my Map" width="600" height="526" class="img-example"/>
```

Comment:
The QR code doesn't look like a real QR code. It's like it's been copied and pasted four times by the looks of the three corner/registration blocks showing up all over the place.

Reply:
Looks good now.